subiquity/models/filesystem.py

```diff
@@ -1041,6 +1041,12 @@ def reset(self):
         self.grub = None
         self.guided_configuration = None
 
+    def get_orig_model(self):
+        orig_model = FilesystemModel(self.bootloader)
+        orig_model.target = self.target
+        orig_model.load_probe_data(self._probe_data)
+        return orig_model
```

Comment:
There is already self._orig_config (which appears to be a copy of the data that has been passed through storage_config.extract_storage_config). Maybe the uses of that can switch to this API too?

Reply:
self._orig_config is in terms of curtin actions, which we are not using on the V2 API.

In this specific case, getting the original partition size, I think that can be worked around by going to orig_config, selecting the object with the matching path, and then getting the size, but this is all a strange request for software that is otherwise expecting a V2-like set of objects, and other similar-sounding queries may be disproportionately harder.

If no client uses the curtin action API I think the existing _orig_config can go away?

I can add a comment talking about this difference.